packages/metal-quartz-icon/src/MetalQuartzIcon.js

```diff
@@ -9,6 +9,8 @@ import templates from './MetalQuartzIcon.soy.js';
 /**
  * Implementation of the Metal Quartz Icon.
  * @see  {@link http://liferay.github.io/lexicon/content/icons-lexicon/|Lexicon Definition}
+ * {@link http://liferay.github.io/lexicon/content/icons-font-awesome/}
+ * {@link http://liferay.github.io/lexicon/content/icons-glyphicons/}
```

Comment:
We can link the Glyphicon documentation directly, but we still use the old version of Font Awesome that we have customized from 6.2. We never migrated to Font Awesome 4.